### ISSUE TITLE
Add function to get length of string without terminal codes

### ISSUE THREAD

**Issue by atommclain** [mtynior/ColorizeSwift#8]
I have been noodling around with a side project using this framework. I really like it!

One thing I have been doing is composing strings across different functions, and certain points I would like to know the length of the string, but without color formatting options.

I put together this janky bit of code to help with that, not sure if it's something you'd like to incorporate into your framework or not.

```
extension NSRegularExpression {
    convenience init(_ pattern: String) {
        do {
            try self.init(pattern: pattern, options: [.caseInsensitive, .allowCommentsAndWhitespace])
        } catch {
            preconditionFailure("Illegal regular expression: \(pattern).")
        }
    }
}

extension String {
    private static let regexPattern = """
    \u{001B}
    \\[
    [0-9]{1,3}
    m
    """
    private static let regex = NSRegularExpression(regexPattern)
    var colorlessCount: Int {
        get {
            let range = NSRange(location: 0, length: self.utf16.count)
            let out = String.regex.stringByReplacingMatches(in: self, options: [], range: range, withTemplate: "")
            return out.count
        }
    }
}
```

**Comment by mtynior** [comment]
Hello there, thanks for using my little library.

I've just published version `1.6.0`. It has the `uncolorized()` method which returns string without any styles:
```
let styledString = "Awful combination".colorize(.yellow, background: .red) // \u{001B}[48;5;9m\u{001B}[38;5;11mAwful combination\u{001B}[0m\u{001B}[48;5;9m\u{001B}[0m
let withoutStyles = styledString.uncolorized() // "Awful combination"
```
The method is more generic, but you can use the `.count` on a uncolorized string to get the length ;)

**Comment by atommclain** [comment]
Very cool; thanks!

That is a great solution that lends itself to more options for future "issues". :D
